Jenkinsfile

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env groovy
 
 docker_oraclejdk8 {
+    dockerRepos = ['confluentinc/kafka-streams-examples']
+    dockerRegistry = '368821881613.dkr.ecr.us-west-2.amazonaws.com/'
```

Comment:
this should probably have some other dns name that we can redirect as we wish

Reply:
Unfortunately, this causes errors due to a principal name mismatch in the certificate :(

Reply:
wow, how terrible and ugly. seems they've had requests for this for a couple of years now and still haven't implemented it.